SuperblocksHQ/superblocks-lab
# Add fullscreen button in appview toolbar

### Description of the Change
![fullscreen1](https://user-images.githubusercontent.com/17637244/52282754-65eb5980-2961-11e9-97df-2ba933cc0101.PNG)
![fullscreen2](https://user-images.githubusercontent.com/17637244/52282753-65eb5980-2961-11e9-8725-508e90fd332c.PNG)

Add new button in appview toolbar which toggles fullscreen of appview




### Benefits
- When you are presenting some DApp, it's nice to have it in larger window
- When you want to test responsivness it's easier this way




### Verification Process
1. Open appview
2. Press fullscreen button
3. Appview should now be toggled in fullscreen
4. Press same button again - appview should be back to normal
